pkg/cable/metrics.go

```diff
@@ -0,0 +1,54 @@
+package cable
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var connectionLabels = []string{
+	// destination clusterID
+	"clusterID",
```

Comment:
let's use cluster_id , private_ip instead of camelCase

Reply:
@maayanf24 @skitt not sure if this is inline with what you spoke yesterday, I wasn't able to attend.

Reply:
the current shared layer use `under_scores` as the labels and metrics names conventions.